backend/functions/orders-manual-sync/orders-manual-sync.function.ts

```diff
@@ -19,23 +21,30 @@ initializeApplicationInsights();
  *
  * curl -v -d '{"txIdOverride": '0'}' -H "Content-Type: application/json" http://localhost:7071/api/orders-sync
  *
- * @param functionContext
+ * @param invocationContext
  * @param request
  */
-const httpTrigger: AzureFunction = async function (
-  functionContext: Context,
+export default async function handler(
   request: HttpRequest,
-): Promise<void> {
-  const context = await ContextCreator.applicationContextCreator(functionContext, request);
+  invocationContext: InvocationContext,
+): Promise<HttpResponseInit> {
+  const logger = ContextCreator.getLogger(invocationContext);
+  const context = await ContextCreator.getApplicationContext({
+    invocationContext,
+    logger,
+    request,
+  });
 
   const ordersController = new OrdersController(context);
   try {
-    const results = await ordersController.syncOrders(context, request.body);
-    functionContext.res = httpSuccess(results);
-  } catch (camsError) {
-    context.logger.camsError(camsError);
-    functionContext.res = httpError(camsError);
+    const results = await ordersController.syncOrders(context, request.body as SyncOrdersOptions);
```

Comment:
We should only be sending the context to the controller, not dealing with the request.